flutter/test/widgets_binding_observer_test.dart

```diff
@@ -400,5 +412,70 @@ void main() {
 
       instance.removeObserver(observer);
     });
+
+    testWidgets('debouncing didChangeMetrics with 100ms delay',
+        (WidgetTester tester) async {
+      final hub = MockHub();
+
+      final observer = SentryWidgetsBindingObserver(
+        hub: hub,
+        options: flutterTrackingEnabledOptions,
+      );
+      final instance = tester.binding;
+      instance.addObserver(observer);
+
+      // ignore: deprecated_member_use
+      final window = instance.window;
+
+      // ignore: deprecated_member_use
+      window.physicalSizeTestValue = window.physicalSize;
+
+      const newPixelRatio = 1.7;
+      // ignore: deprecated_member_use
+      window.devicePixelRatioTestValue = newPixelRatio;
+
+      verifyNever(hub.addBreadcrumb(captureAny));
+
+      //waiting for debouncing with 100ms added https://github.com/getsentry/sentry-dart/issues/400
```

Comment:
nitpick, adding some whitespace :D 

there are more of these cases

```suggestion
      // waiting for debouncing with 100ms added https://github.com/getsentry/sentry-dart/issues/400
```